tests/platform_tests/api/test_chassis_fans.py

```diff
@@ -168,15 +181,32 @@ def test_get_direction(self, duthosts, enum_rand_one_per_hwsku_hostname, localho
 
     def test_get_fans_target_speed(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, platform_api_conn):
 
+        duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+        fans_skipped = 0
+
         for i in range(self.num_fans):
             speed_target_val = 25
+            speed_controllable = self.get_fan_facts(duthost, i, True, "speed", "controllable")
+            if not speed_controllable:
```

Comment:
What happens with platforms that does not support the capability query?  Wouldn't this cause those platforms to be skipped as well?  
Am I missing something perhaps??

Reply:
never mind my comments... I see there is a default value involved with the call so it should be fine...